src/oauth/index.js

```diff
@@ -3,10 +3,22 @@ createNameSpace("realityEditor.oauth");
 import { loadToken } from './tokens.js';
 
 (function(exports) {
+    function getToolboxEdgeBasePath() {
+        const windowPath = window.location.pathname;
+        return '/' + windowPath.split('/').slice(2,6).join('/') + '/i/whud7837yhd'; // => (/stable dropped when connecting to edge-server rather than ui) /n/networkId/s/networkSecret
+    }
+    
     function initService() {
         realityEditor.network.addPostMessageHandler('getOAuthToken', (msgData) => {
-            const { frame, urls, clientId, clientSecret } = msgData;
-            loadToken(frame, urls, clientId, clientSecret).then(token => {
+            const { frame, authorizationUrl, clientId } = msgData;
+            const object = Object.values(realityEditor.objects).find(obj => {
+                return Object.keys(obj.frames).includes(frame);
+            });
+            // The edge server that will handle the processing of the auth code to receive an auth token
+            const edgeServer = window.location.origin.includes('toolboxedge') ? window.location.origin + getToolboxEdgeBasePath() : `http://${object.ip}:${object.port}`;
```

Comment:
It would be good for all of these window.location.origin.includes() to be refactored out to a single IS_CLOUD variable to make it easier for the Atlas-hosted version of this to rewrite toolboxedge to something else